src/EditorFeatures/CSharp/FixInterpolatedVerbatimString/FixInterpolatedVerbatimStringCommandHandler.cs

```diff
@@ -35,6 +35,20 @@ public FixInterpolatedVerbatimStringCommandHandler()
         public string DisplayName => CSharpEditorResources.Fix_interpolated_verbatim_string;
 
         public void ExecuteCommand(TypeCharCommandArgs args, Action nextCommandHandler, CommandExecutionContext executionContext)
+        {
+            try
+            {
+                ExecuteCommandWorker(args, nextCommandHandler, executionContext);
+            }
+            catch (OperationCanceledException)
+            {
+                // According to Editor command handler API guidelines, it's best if we return early if cancellation
+                // is requested instead of throwing. Otherwise, we could end up in an invalid state due to already
+                // calling nextCommandHandler().
+            }
+        }
+
+        private static void ExecuteCommandWorker(TypeCharCommandArgs args, Action nextCommandHandler, CommandExecutionContext executionContext)
         {
             // We need to check for the token *after* the opening quote is typed, so defer to the editor first
             nextCommandHandler();
```

Comment:
📝 Immediately after this call, the code should check `executionContext.OperationContext.UserCancellationToken`. If cancellation is requested but `nextCommandHandler()` didn't throw, we should just return immediately.